test/membrane_flv_plugin/demuxer_test.exs

```diff
@@ -87,6 +84,6 @@ defmodule Membrane.FLV.Demuxer.Test do
     video = File.read!("/tmp/video.h264")
 
     assert byte_size(audio) == 96_303
-    assert byte_size(video) == 144_615
+    assert byte_size(video) == 144_617
```

Comment:
the previous NALu prefix was `<<0, 0, 1>>` and the new parser uses `<<0, 0, 0, 1>>`